test/index.spec.ts

```diff
@@ -0,0 +1,155 @@
+import { rules as typescriptEslintRules } from '@typescript-eslint/eslint-plugin';
+import { CLIEngine, Linter } from 'eslint';
+import { rules as arrayFuncRules } from 'eslint-plugin-array-func';
+import { rules as eslintCommentsRules } from 'eslint-plugin-eslint-comments';
+import { rules as immutableRules } from 'eslint-plugin-immutable';
+import { rules as jestRules } from 'eslint-plugin-jest';
+import { rules as simpleImportSortRules } from 'eslint-plugin-simple-import-sort';
+import { rules as unicornRules } from 'eslint-plugin-unicorn';
+
+import * as ourRules from '../rules';
+import * as fixturesPromises from './fixtures';
+
+/*
+ * Test utilities
+ */
+
+interface VerifyResponse {
+  [key: string]: unknown;
+  warningCount: number;
+  messages: [];
+  errorCount: number;
+}
+
+const cli = new CLIEngine();
+const linter = new Linter();
+
+const eslintRules = Object.fromEntries(linter.getRules());
+
+const ourRulesToOriginalMap = {
```

Comment:
Our? Who is we?

Reply:
fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/267a96360f447175bf63c69e8f81d71b5db5c652